include/grpcpp/impl/codegen/completion_queue.h

```diff
@@ -41,6 +41,10 @@
 
 struct grpc_completion_queue;
 
+namespace grpc_impl {
+
+class ServerBuilder;
```

Comment:
Maybe try without the forward declaration?

Reply:
I tried removing it.. throws a compile error.. probably better with the forward declaration that include as this is codegen..